pkg/filesystem/fs/fs.go

```diff
@@ -336,6 +336,16 @@ func (fs *Filesystem) RemoveCache(blobDigest string) error {
 		return errors.Wrapf(err, "invalid blob digest from label %s", label.CRILayerDigest)
 	}
 	blobID := digest.Hex()
+	if fs.fsDriver == config.FsDriverFscache {
+		c, err := fs.getSharedDaemon().GetClient()
+		if err != nil {
+			log.L.Warnf("GetClient err %s", err)
+		}
+		// delete fscache blob cache file
+		if err := c.UnbindBlob("", blobID); err != nil {
+			log.L.Warnf("delete blob %s err %s", blobID, err)
+		}
+	}
 	return fs.cacheMgr.RemoveBlobCache(blobID)
```

Comment:
For fscache driver, skip `RemoveBlobCache` ?

Reply:
We also need to delete meta and chunkmap files in RemoveBlobCache. so do we need let nydusd do this for fscache?

Reply:
It has no effect. Fuse driver and fscache driver have different places to put blog meta files which won't be deleted by the function RemoveBlobCache now...

Reply:
Nydus-snapshotter cache manager is somewhat conflicting with fscache driver

Reply:
Understood , will fix this.